pegasus/helpers/section_api_helpers.rb

```diff
@@ -372,6 +381,17 @@ def to_hash()
     }
   end
 
+  def to_shallow_hash()
+    {
+        id:@row[:id],
```

Comment:
Two things with this:

1. Could remove some duplicate code by making the implementation of `to_hash` be `to_shallow_hash.merge(...additional properties ...)`

2. It appears the distinction between these two methods is not whether it's 'shallow' or 'deep', but is a roles/permissions decision between what's accessible to the section owner vs the section member. In the future one might imagine that there are 'deep' fields that are accessible to members and shallow fields that are accessible to owners only. Suggest changing the names of these methods (something like to_owner_hash/to_member hash or to_teacher_hash/to_student_hash?).